websites/G/Google Fonts/presence.ts

```diff
@@ -0,0 +1,79 @@
+const presence = new Presence({
+    clientId: "814919836835905566"
+}),
+tmb = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "fontsred",
+        smallImageKey: "google",
+        smallImageText: "Google",
```

Comment:
really?

Reply:
I did this on my Google Messages presence. I think it's not cool when there isn't any text on the small image even if it's obvious... Do I really need to remove it ?

Reply:
Yes, you do.